config-linux.md

```diff
@@ -642,6 +642,7 @@ The values MUST be absolute paths in the [container namespace](glossary.md#conta
 ## <a name="configLinuxMountLabel" />Mount Label
 
 **`mountLabel`** (string, OPTIONAL) will set the Selinux context for the mounts in the container.
+    The value MUST NOT be an empty string.
```

Comment:
> Probably something equivalent to `an empty value will not be applied / have no effect` is good enough.

Runtimes [don't have to validate this if they don't want][1].  But when we *are* validating, I think we want to treat empty-string `mountLabel`s as errors for the same reasons that we treat `null`s as errors in a number of places (see [here][2] and #662).  Why would a config author who [understood that empty strings are ignored][3] set an empty-string value instead of leaving the property unset?  I think it's much more likely that config authors explicitly setting an empty-string value actually expect it to do something, and a “that's not valid” error will help correct their misunderstanding.


[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc5/runtime.md#L109
[2]: https://github.com/opencontainers/runtime-spec/pull/555#issuecomment-272020515
[3]: https://github.com/opencontainers/runtime-spec/pull/843#issuecomment-303248848